ruby-challenges.rb

```diff
@@ -3,17 +3,57 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
+#INPUT:  Variables num1 = 7 ;  num2 = 42  ; num3 = 221
+
+#OUTPUT:  '7 is odd' ; '42 is even'  ; '221 is odd'       
+                    ######      Determining if an integer is even or odd, my memory bring me back to using conditional 
+                    #      statements to output the correct answer. This involves modulo two is equal to zero for even, and
+                    # the same operation including the bang operator exclimation point. I chose to return the even number with
+                    # an IF statement, and the else IF with the ODD outcome. 
+                              # Although very straighforward, the terminal exclaims back with an "unexpected end-of-input", 
+                    # highlighting the modulo operator. I'm pretty stuck after moving some syntax around          
+
+
+
 num1 = 7
 # Expected output: '7 is odd'
 num2 = 42
 # Expected output: '42 is even'
 num3 = 221
 # Expected output: '221 is odd'
 
+        def which_nums(num1, num2, num3)
```

Comment:
Your logic is heading in the right direction, but consider how you can make this more dynamic.  Functions and methods both are meant to make it so we can reuse the code block for ANY values passed in.